pkgs/tools/security/cosign/default.nix

```diff
@@ -21,13 +21,15 @@ buildGoModule rec {
 
   subPackages = [ "cmd/cosign" ];
 
-  buildFlagsArray = [ "-ldflags=-s -w -X github.com/sigstore/cosign/cmd/cosign/cli.gitVersion=${version}" ];
+  preBuild = ''
+    buildFlagsArray+=("-ldflags" "-s -w -X github.com/sigstore/cosign/cmd/cosign/cli.gitVersion=v${version}")
```

Comment:
Is there a difference with `buildFlagsArray`?

It seems the documentation recommends its usage:
https://github.com/NixOS/nixpkgs/blob/80b395015b7813cd9adf3ae2d50db15453485bef/doc/languages-frameworks/go.section.md#L115-L132

Reply:
See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md

Reply:
Thanks for the explanation, should not the Go documentation be updated?

Reply:
Yes. It should be.